app/views/search/search_jobs.html.erb

```diff
@@ -23,4 +23,27 @@
   </div>
 </div>
 
-<%= render partial: 'search_footer' unless request.xhr? %>
+<!-- Navigation -->
+<div class="navigation">
+  <div class="row">
+    <div class="small-12 columns">
+      <ul class="small-block-grid-1 medium-block-grid-3 cards small">
+        <li>
+          <a href="https://veterans.linkedin.com/">
+            <h5>LinkedIn</h5>
+          </a>
+        </li>
+        <li>
+          <a href="https://www.surveymonkey.com/r/QSGQRFN">
+            <h5>Coursera</h5>
+          </a>
+        </li>
+        <li>
+          <a href="/employment/job-seekers/create-resume">
+            <h5>Resume Builder</h5>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
```

Comment:
This whole footer feels out of place and like I don't know what the links are there for. Can we either provide more context for these links or remove them @emilyville ?